### PR TITLE
Return information_schema schema for doltgres

### PR DESCRIPTION
**Pull request by tbantle22**
GMS PR: https://github.com/dolthub/go-mysql-server/pull/2678

### CODE REVIEW

File: go/libraries/doltcore/sqle/database.go [hunk 1]
```diff
@@ -1377,6 +1378,10 @@ func (db Database) CreateSchema(ctx *sql.Context, schemaName string) error {
 
 // GetSchema implements sql.SchemaDatabase
 func (db Database) GetSchema(ctx *sql.Context, schemaName string) (sql.DatabaseSchema, bool, error) {
+	if schemaName == sql.InformationSchemaDatabaseName {
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/database.go:1381]: Would add a note about this only being relevant for Doltgres

File: go/libraries/doltcore/sqle/database.go [hunk 2]
```diff
@@ -1443,9 +1448,59 @@ func (db Database) AllSchemas(ctx *sql.Context) ([]sql.DatabaseSchema, error) {
 		dbSchemas[i] = handledDb
 	}
 
+	dbSchemas[len(schemas)] = newInformationSchemaDatabase(db.Name())
+
 	return dbSchemas, nil
 }
 
+type informationSchemaDatabaseSchema struct {
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/database.go:1456]: Would put this type into its own file, this one is pretty crowded already

**Review by zachmu — APPROVED** [review]
LGTM

**Comment by coffeegoddd** [comment]
@tbantle22 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>76db2ba</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>76db2ba</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**